lib/modelMaterialsCommon.js

```diff
@@ -558,16 +558,18 @@ function generateTechnique(gltf, khrMaterialsCommon, lightParameters) {
     // Add shaders
     shaders[vertexShaderId] = {
         type: WebGLConstants.VERTEX_SHADER,
-        uri: 'data:,' + vertexShader,
         extras: {
-            source: vertexShader
+            _pipeline : {
+                source : vertexShader
+            }
```

Comment:
This step happens after `loadGltfUris`, and after that point the pipeline doesn't really look in the uri.